hazelcast/util.py

```diff
@@ -175,6 +135,21 @@ def get_portable_version(portable, default_version):
     return version
 
 
+def deserialize_list_in_place(data_list, to_object_fn):
```

Comment:
Should there be type annotation for the new functions?

Reply:
That makes sense, I wish we fully type hint the code base so that we can add checks for such things.

Reply:
https://github.com/hazelcast/hazelcast-python-client/pull/618/commits/c8ff71bc35d74b6655c8125ac7d814a00c79814c